src/status_im/ui/screens/browser/events.cljs

```diff
@@ -133,6 +140,13 @@
        (and (= type constants/history-state-changed) platform/ios? (not= "about:blank" url))
        (model/update-browser-history-fx browser url false cofx)
 
+       (= type constants/scan-qr-code)
+       {:dispatch [:qr-scanner.ui/scan-qr-code-pressed
```

Comment:
Why dispatch instead of just calling `qr-scanner/scan-qr-code` and returning its effects ?